src/components/toastNotification/view/toastNotificaitonView.js

```diff
@@ -1,6 +1,6 @@
 import React, { Component } from 'react';
 import { TouchableOpacity, Text } from 'react-native';
-import Animated, { Easing } from 'react-native-reanimated';
+import * as Animatable from 'react-native-animatable';
```

Comment:
better use import { View as AnimatableView } from 'react-native-animatable' import * .... is almost always very expensive import

Reply:
Ok